Swift/BRCrypto/BRCryptoWalletManager.swift

```diff
@@ -559,6 +559,9 @@ extension WalletManager {
             funcGetBlockNumber: { (context, cwm, sid) in
                 let manager = Unmanaged<WalletManager>.fromOpaque(context!).takeUnretainedValue()
                 precondition (nil != cwm, "SYS: BTC: GetBlockNumber: Missed {cwm}")
+                defer {
+                    if let cwm = cwm { cryptoWalletManagerGive(cwm) }
+                }
```

Comment:
FYI: Since the `precondition()` confirms the `cwm` is not `nil`, the `defer` can be `defer { cryptoWalletManagerGive (cwm!) }`

Reply:
I think CORE-392 will eventually provide `precondition(nil != cwm)` for all/most-of the cases below.

Reply:
I added the precondition check throughout, since it should never be null for these callbacks.

After more thought, I've also moved the release of CWM to the completion handler.

Reply:
Good catch on the completion handler.